### PR TITLE
Add hamburger-style menu for mobile viewing of calendar/categories sidebar

### PR DESCRIPTION
**Pull request by michaelsilver**
Still needs transitions and has a known white flash bug on iOS Safari

kapture 2019-03-03 at 20 45 54

### CODE REVIEW

File: src/components/home/HomeView.css [hunk 1]
```diff
@@ -33,12 +33,36 @@
...
+
+.HomeView .left .greybox.active {
+  position: absolute;
+  z-index: 1;
+  display: block;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: #80808080;
```
Comment (jackcook) [inline src/components/home/HomeView.css]: I think you meant #808080

**Comment by jackcook** [comment]
Also make sure to resolve the merge conflict

File: src/components/home/HomeView.css (hunk 1 above)
Comment (michaelsilver) [reply src/components/home/HomeView.css]: Nah the last 80 is the transparency

Comment (jackcook) [reply src/components/home/HomeView.css]: Wait I'm so shook I've never seen this before

**Pull request merged**